fusionauth/resource_fusionauth_tenant_helpers.go

```diff
@@ -515,13 +515,15 @@ func buildResourceDataFromTenant(t fusionauth.Tenant, data *schema.ResourceData)
 		return diag.Errorf("tenant.name: %s", err.Error())
 	}
 
-	err = data.Set("oauth_configuration", []map[string]interface{}{
-		{
-			"client_credentials_access_token_populate_lambda_id": t.OauthConfiguration.ClientCredentialsAccessTokenPopulateLambdaId,
-		},
-	})
-	if err != nil {
-		return diag.Errorf("tenant.oauth_configuration: %s", err.Error())
+	if lambdaID := t.OauthConfiguration.ClientCredentialsAccessTokenPopulateLambdaId; lambdaID != "" {
```

Comment:
This was funky..
If the field was empty, Terraform would report having to report having to remove a blank object under test:

```diff
- oauth_configuration: {}
```